### ISSUE TITLE
a problem in training process when  i use aptardi

### ISSUE THREAD

**Issue by haiyonghe** [luskry/aptardi#6]
Hello! 
I use the demo data when i want to build my model, It always reminds me of a mistake。
maybe there is something wrong with class_weight in model.fit()?
by the way, I sure I have installed right version packages.
Thank you for you help 
Waiting for your reply
-------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/bin/aptardi", line 1974, in <module>
    main()
  File "/usr/local/bin/aptardi", line 1934, in main
    history = model.fit(X_train, y_train, validation_split = 0.25, epochs = 25, verbose = 0, class_weight = class_weights, callbacks = [model_cp])
  File "/home/ubuntu/anaconda3/envs/py3.7.7/lib/python3.7/site-packages/tensorflow/python/keras/engine/training.py", line 1064, in fit
    steps_per_execution=self._steps_per_execution)
  File "/home/ubuntu/anaconda3/envs/py3.7.7/lib/python3.7/site-packages/tensorflow/python/keras/engine/data_adapter.py", line 1116, in __init__
    if class_weight:
ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()

**Comment by luskry** [comment]
Thank you for bringing this to our attention.

This was due to upgrading tensorflow to 2.4.0 instead of 2.0.0 after a bot recommendation. However, it turns out the class_weight no longer works as before.

We were able to make a model work by adding sample_weight_mode = 'temporal' to model.compile(), changing the masking weights to 0 instead of NA, and changing class_weight to sample_weight in model.fit; however, the model training changed.

As a result, we changed the requirements back to tensforflow 2.0.0. Note that Python versions 3.8.x and up are not compatible with this version of tensorflow, so you will need older versions.

TL;DR please use tensorflow 2.0.0 instead of 2.4.0. This version of tensorflow requires Python 3.7.x versions or OLDER. 

Please let us know if this fixes the problem.

**Comment by haiyonghe** [comment]
Thank you luskry!
Change tensorflow version to 2.0.0 really fixes the problem.
